.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   - "4"
 
-sudo: false
```

Comment:
Seems like, but it fixes downloading chrome on ubuntu images: https://github.com/travis-ci/travis-ci/issues/5899

If there's another way to do it I'm all for it, tried this out and it worked so I figured it was good for now.